CentralManagement/ChocoCCM Command Reference/GetCCMOutdatedPackageMember.md

```diff
@@ -0,0 +1,82 @@
+# Get-CCMOutdatedPackageMember
+
+<!-- This documentation is automatically generated from /Get-CCMOutdatedPackageMember.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+Returns computers with the requested outdated software
+
+## Syntax
+
+~~~powershell
+Get-CCMOutdatedPackageMember `
+  [-Software <String>] `
+  [-Package <String>] [<CommonParameters>]
+~~~
```

Comment:
None of this doc really clarifies what the difference is in using `-Software` vs `-Package` here IMO, and the cmdlet name is equally opaque. Consider renaming the command or parameters & expanding the explanation in the doc.

IMO we should use `-PackageId` and then `-Name` or `-FriendlyName` instead of `-Software` which is too ambiguous.

The `Member` suffix in this command name feels unnecessary.

Reply:
Renamed this to `Get-CCMOutdatedSoftwareMember. The API endpoints are _different_, so `Get-CCMOutdatedSoftware will list all of your softwares that are marked as Outdated, and this cmdlet will show you what computers that outdated software is installed on. It's a bit messy, but that's how the API is at the minute.